opentracing-shim/src/test/java/io/opentelemetry/opentracingshim/testbed/clientserver/TestClientServerTest.java

```diff
@@ -66,12 +66,15 @@ private void verify() {
 
     assertThat(finished.get(1).getTraceId()).isEqualTo(finished.get(0).getTraceId());
     Kind firstSpanKind = finished.get(0).getKind();
-    if (firstSpanKind == Kind.CLIENT) {
-      assertThat(finished.get(1).getKind()).isEqualTo(Kind.SERVER);
-    } else if (firstSpanKind == Kind.SERVER) {
-      assertThat(finished.get(1).getKind()).isEqualTo(Kind.CLIENT);
-    } else {
-      fail("Unexpected first span kind: " + firstSpanKind);
+    switch (firstSpanKind) {
```

Comment:
Enum swith, not if/else

Reply:
Honestly for just two values plus default, I think if/else is slightly more readable. But on the other hand, enforced errorprone-enforced consistency may still be better 🤷